integrations/src/test/java/io/jenkins/plugins/casc/ToolDefaultPropertiesExportBlacklistTest.java

```diff
@@ -26,7 +26,7 @@ public void export_tool_configuration() throws Exception {
         ConfigurationContext context = new ConfigurationContext(registry);
         CNode yourAttribute = getToolRoot(context);
 
-        String exported = toYamlString(yourAttribute);
+        String exported = toYamlString(yourAttribute).replaceAll("git\\.exe", "git");
```

Comment:
@timja you evil person 🤣

Reply:
bahahaha, really should be fixed in git plugin somehow 😉 
annoying that it configures a default for you

Reply:
A simple `replace("git.exe", "git")` would have been enough here, no need for a regular expression ;)

Reply:
@jansohn feel free to send a PR 😅